src/main/java/com/kilopo/kosshop/entity/Order.java

```diff
@@ -0,0 +1,75 @@
+package com.kilopo.kosshop.entity;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import java.util.Date;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Table(name = "orders")
+public class Order extends BaseEntity {
+
+    @Column(nullable = false)
+    private double sum;
+
+    @Column(nullable = false)
+    private Date date;
+
+    @Enumerated(EnumType.STRING)
+    private OrderStatus orderStatus;
+
+    @ManyToOne
+    private User user = new User();
+
+    @OneToMany()
```

Comment:
remove (), and add here @JoincColumn i think better add column to  OrderProduct than creating a new table